dradis-pentera.gemspec

```diff
@@ -17,6 +17,7 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency 'dradis-plugins', '~> 4.0'
 
-  s.add_dependency 'combustion', '~> 0.6.0'
-  s.add_dependency 'rspec-rails'
+  spec.add_development_dependency 'rspec-rails'
+  spec.add_development_dependency 'combustion', '~> 0.5.2'
```

Comment:
are we using this?

Reply:
Yes both are needed for the standalone specs